### ISSUE TITLE
String produces Fleece & Shears in uncrafter

### ISSUE THREAD

**Issue by Sanjurjo7** [DarkPacks/SevTech-Ages#4588]
## Bug Description
String produces Fleece & shears in Uncrafter

### Did this behavior use to work in the previous version?
Not sure

## Steps to Reproduce (for bugs)

1. Place string in uncrafter
2. Select outputs, can vary shears
3. Profit

## Logs
n/a

## Client Information
* Modpack Version:3.2.1
* Java Version:8JDK Linux
* Launcher Used:MultiMC
* Memory Allocated:2048-4096
* Server/LAN/Single Player:Single Player
* Optifine Installed:N/A
* Shaders Enabled:None

## World Information
* Modpack Version world created in:3.2.1
* Additional Content Installed:Better foliage and fancy block break disabled.

**Comment by Sanjurjo7** [comment]
Posting because this seems imbalanced.

**Comment by eturner** [comment]
Can confirm. Just tried it, got superium shears. You need more than one string in the uncrafter.
P.S. pls don't take my uncrafter

**Comment by sam-kirby** [comment]
Assuming this issue refers to the Twilight Forest Uncrafting Table:

There is no configuration for this block. I don't know why it was left enabled when a similar block from Cyclic was disabled for similar reasons to this (see #2436).

The only reasonable way to resolve this is to disable the block.

**Comment by eturner** [comment]
Yeah, the Uncrafting Table from Twilight Forest. I'll have no idea how to get all the diamonds I need after it's gone
